### DISCUSSION TITLE
Several paths in executable option

### DISCUSSION THREAD

**Discussion by jowe19** [tndrle/PyLuaTeX#23] [Ideas]
Hello, 
we use `pyluatex` to compile a book for several colleagues. Except that each colleague would need to set their own `executable =` option.

Is there an option to ask pyluatex to try several paths for python?

**Comment by tndrle** [comment]
There is no such option.

Can you construct such functionality with `\IfFileExists`?

For example
```latex
\newcommand*{\setexec}[1]{
  \IfFileExists{#1}{\PassOptionsToPackage{executable=#1}{pyluatex}}{}
}
\setexec{/usr/bin/python}
\setexec{/usr/bin/python3}
% \setexec{...} % and so on for even more paths to Python

\documentclass{article}
\usepackage{pyluatex}

\begin{document}
\py{1+2}
\end{document}
```
